src/OrchardCore.Modules/OrchardCore.Diagnostics/ViewModels/HttpStatusCodeShapeViewModel.cs

```diff
@@ -0,0 +1,19 @@
+using System.Net;
+using OrchardCore.DisplayManagement.Views;
+
+namespace OrchardCore.Diagnostics.ViewModels;
+
+public class HttpStatusCodeShapeViewModel : ShapeViewModel
+{
+    public HttpStatusCode HttpStatusCode { get; }
+
+    public HttpStatusCodeShapeViewModel(HttpStatusCode statusCode)
+    {
+        HttpStatusCode = statusCode;
+        Metadata.Type = "HttpStatusCode";
```

Comment:
Can we have a common type (`Error`, `HttpError`) and use `_NotFound`, `_404` -> `HttpError.404.cshtml` and `HttpError.NotFound.cshtml`

Reply:
done!